src/leveldbwrapper.h

```diff
@@ -168,6 +190,19 @@ class CLevelDBWrapper
     {
         return pdb->NewIterator(iteroptions);
     }
+
+    /**
+     * Return true if the database managed by this class contains no entries.
+     */
+    bool IsEmpty()
+    {
+        leveldb::Iterator* it = NewIterator();
```

Comment:
Please use a `boost::scoped_ptr<leveldb::Iterator>` here, like in `txdb.cpp`, to automatically free the iterator when it goes out of scope.